utils/awesome-client

```diff
@@ -5,10 +5,7 @@
 # when used with "|tail".
 set -o pipefail
 
-tty 2>&1 > /dev/null
-ISATTY=$?
-
-if [ "$ISATTY" = 0 ]
+if [ -t 1 ]  # is a tty.
```

Comment:
Before: Is NOT a tty
After: Is a tty

I think this line should be `if [ ! -t 1 ]`